### ISSUE TITLE
Better documentation for Credentials.expiryWindow

### ISSUE THREAD

**Issue by rawpixel-vincent** [aws/aws-sdk-js#4258]
### Describe the issue

Link to docs: http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Credentials.html

The entry on expiryWindow just shows the default value of 15. From reading the source code, I think this means 15 seconds. The description should probably say something like "the amount of time (in seconds) before the expireTime that the credentials is considered expired."

### Links

http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Credentials.html

**Comment by ajredniwja** [comment]
Hi @rawpixel-vincent thanks for opening this issue, looking at https://github.com/aws/aws-sdk-js/blob/307e82673b48577fce4389e4ce03f95064e8fe0d/lib/credentials.js#L85 it seems like it is in seconds.

**Comment by aBurmeseDev** [comment]
Checking in here - thanks again for your feedback. We've [confirmed](https://github.com/aws/aws-sdk-js/blob/307e82673b48577fce4389e4ce03f95064e8fe0d/lib/credentials.js#L85) that default value for `expiryWindow` is 15 seconds. However, we have decided not to update the documentation for the v2 version of the SDK at this time, as the information about the default `expiryWindow` value has already been included in the v3 documentation.

v3 Reference: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-credential-providers/#createCredentialChain
![Screenshot 2024-08-27 at 12 06 11 AM](https://github.com/user-attachments/assets/e680298e-a063-4bf0-a4b8-dd31cbd8c321)

Closing the issue. 
Best,
John
